src/services/__tests__/messagesService.test.ts

```diff
@@ -111,7 +119,7 @@ const problemJson = {
 const mockedUser: User = {
   created_at: 1183518855,
   family_name: "Garibaldi",
-  fiscal_code: validFiscalCode as FiscalCode,
+  fiscal_code: aValidFiscalCode as FiscalCode,
```

Comment:
I suggest to declare `const aValidFiscalCode: FiscalCode` above, so you don't have to cast it every time you use it (same for the other fixtures)